### PR TITLE
BASK54: Unit Tests for Searching Profiles #118

### PR DESCRIPTION
**Pull request by TheGreatMarkus**
BASK54 (Issue #118): Added Search unit tests
   - Renamed MenuBarPresenter method
   - Added unit tests for method getAllUsers from MenuBarPresenter
   - Added unit tests for method searchByUsername from MenuBarPresenter
   - Commented out printStackTrace from CrudTests

### CODE REVIEW

File: src/test/java/com/basilisk/SearchTests.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        assertDoesNotThrow(() -> {
+            List<User> allUsers = menuBarPresenter.getAllUsers();
+        });
+        List<User> allUsers = menuBarPresenter.getAllUsers();
+
+        assertNotNull(allUsers);
+
+        assertTrue(allUsers.contains(testUser));
+
+        userService.removeUser(testUser);
```
Comment (Pwnagebird) [inline src/test/java/com/basilisk/SearchTests.java]: Dont need to remove the user at end

File: src/test/java/com/basilisk/SearchTests.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        });
+        // testUser must be in the list if its username contains the search term
+        assertTrue(menuBarPresenter.searchByUsername("qwertyuiop").contains(testUser));
+        assertTrue(menuBarPresenter.searchByUsername("ertyui").contains(testUser));
+
+        // testUser won't be in the list otherwise
+        assertFalse(menuBarPresenter.searchByUsername("asdfghjkl").contains(testUser));
+
+        userService.removeUser(testUser);
+    }
```
Comment (Pwnagebird) [inline src/test/java/com/basilisk/SearchTests.java]: same for here.

Comment (TheGreatMarkus) [reply src/test/java/com/basilisk/SearchTests.java]: Removed those statements

File: src/test/java/com/basilisk/SearchTests.java (hunk 1 above)
Comment (TheGreatMarkus) [reply src/test/java/com/basilisk/SearchTests.java]: Done, removed that line of code.

**Pull request merged**